### ISSUE TITLE
Recommendation: turn `eris init` into `eris`

### ISSUE THREAD

**Issue by taoeffect** [eris-ltd/eris-cli#255]
I think `eris` would do well to follow as best it can the conventions that have been established by other tools, simply for the reason that it will make `eris` easier for newcomers to learn and will be less likely to confuse or turn them off.

Many commonly used tools have an `init` command (just about every VCS?). Typically, `[tool] init` is a command that is meant to be used on a per-project basis (usually this turns out to be per-directory).

What do other tools do when they need to setup a complicated environment? The established convention seems to be the one that `geth` follows:

1. When the command is run, however it is run, it displays informative text (unless a `--yes` was passed).
2. Often, this text is accompanied by a request for user input (usually a `y/n` question), such as "Proceed" or "Agree".

After this intro, it does the setup, and never shows the intro again (unless the essential files are missing).

So, to summarize:

- Following established conventions is good UX.
- `init` has a very established convention, one that does not fit with `eris init`.
- The behavior of `eris init` has a different established convention, one that I'd recommend adopting as it would improve UX.

**Comment by compleatang** [comment]
This is not the convention we're going to follow.

**Comment by compleatang** [comment]
To be more precise eris is gonna go to the help menu not the init sequence. If another synonym for init is preferable I'd be fine discussing that in another issue.
